BlackNectar/BlackNectar/FilterViewController.swift

```diff
@@ -216,16 +263,30 @@ extension FilterViewController {
     
 }
 
-//MARK: Loads Stores When User Pans
+//MARK: Style Menu Code
 extension FilterViewController {
     
-    func mapView(_ mapView: MKMapView, regionDidChangeAnimated animated: Bool) {
+    func styleMenu() {
         
-        let center = mapView.centerCoordinate
+        farmersMarketsButton.layer.borderColor = UIColor.white.cgColor
+        farmersMarketsButton.layer.borderWidth = 1
+        farmersMarketsButton.layer.cornerRadius = 7
         
-        LOG.debug("User dragged Map Screen to: \(center)")
+        groceryStoresButton.layer.borderColor = UIColor.white.cgColor
+        groceryStoresButton.layer.borderWidth = 1
+        groceryStoresButton.layer.cornerRadius = 7
         
-        self.loadStoresInMapView(at: center)
+    }
+    
+    func styleButtonOn(button: UIButton) {
+        
+        button.layer.backgroundColor = UIColor.init(red: 0.902, green: 0.73, blue: 0.25, alpha: 1).cgColor
```

Comment:
These color values seem guessed at. You definitely want to use code that lets you put in colors using HEX value.

Check out https://github.com/RedRoma/RedRomaColors:

https://github.com/RedRoma/RedRomaColors/blob/develop/RedRomaColors/RedRomaColors/Colors.swift

`Colors.fromRGB()`
`Colors.from(hexString:"")`